source/stylesheets/site.css.scss

```diff
@@ -1,8 +1,15 @@
+@import 'slider';
+
 .hidden {
   display: none;
 }
 
+.container {
+  width: 60%;
```

Comment:
Keeping this a nice easy to use size :+1:
Might want to add a media query for portrait orientations since this width only rule will cause the slider to be below the fold on mobile devices that are in portrait mode, but not a blocker.